providers/universal-provider/src/UniversalProvider.ts

```diff
@@ -87,12 +89,13 @@ export class UniversalProvider implements IUniversalProvider {
 
   public sendAsync(
     args: RequestArguments,
-    callback: (error: Error | null, response: any) => void,
+    callback: (error: Error | null, response: JsonRpcResult) => void,
     chain?: string | undefined,
   ): void {
+    const id = new Date().getTime();
     this.request(args, chain)
-      .then((response) => callback(null, response))
-      .catch((error) => callback(error, undefined));
+      .then((response) => callback(null, formatJsonRpcResult(id, response)))
```

Comment:
>.then((response) => callback(null, formatJsonRpcResult(id, response)))

Do we need to format the response here? Afaik all responses are already formatted as `jsonRpcResult` by the responding client

Reply:
Yes, the client is not responding in a formatted manner

Reply:
please make sure you run

- `npm i` in root to fetch new packages
- `npm run reset` to rebuild the monorepo
for some reason the relay is rejecting your jwt and/or projectId

also your system's datetime as jwt is valid for 1 day so if you datetime is forward/back in time might cause it

Reply:
Hey @ganchoradkov, I tried that also and ran it in a local file but it gave the **same error**. For passing the test case there might be a change in the workflow file of the repo where it is not getting environment variables correctly. 

If possible can I DM you on Discord or any other platform to resolve this issue?

Reply:
`@GanchoRadkov` on telegram